app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -0,0 +1,57 @@
+module Users
+  class PivCacAuthenticationSetupController < ApplicationController
+    include UserAuthenticator
+
+    before_action :confirm_two_factor_authenticated
+    before_action :authorize_piv_cac_setup, only: :new
+    before_action :authorize_piv_cac_disable, only: :disable
+
+    def new
+      analytics.track_event(Analytics::USER_REGISTRATION_PIV_CAC_SETUP_VISIT)
+      if params.key?(:token)
+        process_token(params[:token])
```

Comment:
It looks like this part acts more like what I would expect to see in a `create` action. For example, when setting up TOTP, we show the user a page (the `new` action) where they can then submit a form to complete the setup. 

These are 2 separate events for analytics: one for the page view, and one for submitting the form. When submitting a form, we want to capture more than just the fact that the form was submitted. If you look at our other controllers, the pattern is something like this:
```ruby
result = Form.submit(params)
analytics.track_event(EVENT_NAME, result.to_h)
```
where `result` is a `FormResponse` object.

Reply:
Good point. I'll make it two different events.

Reply:
Forgot to do that before the squash/rebase. Will add another commit with that.